dlp/README.rst

```diff
@@ -61,38 +63,15 @@ Windows
     <your-env>\Scripts\activate
     <your-env>\Scripts\pip.exe install google-cloud-dlp
 
-Preview
```

Comment:
I can adjust the synth script to not move this file. this example might be useful.